src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -4,6 +4,6 @@
     <title>My team</title>
 </head>
 <body>
-<h1>Hello, mates!</h1>
+    <h1>Hello, mates!</h1>
```

Comment:
As improvements - it's a good idea to create a menu from the index page to other pages, cause users might not know which menu your application has.